egs/wsj/s5/steps/nnet3/dump_bottleneck_features.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
For the sake of consistency with naming convention used in Kaldi, use the name
`steps/nnet3/make_bottleneck_features.sh`.